### DISCUSSION TITLE
My GitHub page doesn't show up

### DISCUSSION THREAD

**Discussion by kevinwd2401** [community/community#28082] [Other Feature Feedback, Questions, & Ideas] [answered]
hi, my GitHub page doesn't show up even though the website said it was live 18 days ago. I have googled a lot on the common causes but haven't found a solution. Can someone please help? Thank you! My repository is at https://github.com/kevinwd2401/kevinwd2401.github.io

![image](https://user-images.githubusercontent.com/109395354/182419552-27f77a1c-5c1a-49da-8fa7-c28a138fd385.png)

**Comment by epistax1s** [comment, marked as answer]
Try changing the file name from `Index.html `on `index.html` (Using only lowercase characters.).

[From the documentation:](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site)

> Create the entry file for your site. GitHub Pages will look for an `index.html`, `index.md`, or `README.md` file as the entry file for your site.

**Reply by kevinwd2401** [reply]
Once I changed the spelling of the file name to index.html and it started working! Thank you so much! It was such a trivial thing but I was stuck for days! Really appreciate your help!
